x-pack/test/security_solution_endpoint/apps/endpoint/policy_details.ts

```diff
@@ -493,7 +493,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
                 version: policyInfo.packageInfo.version,
               },
             },
-            artifact_manifest: agentFullPolicy.inputs[0].artifact_manifest,
+            artifact_manifest: agentFullPolicyUpdated.inputs[0].artifact_manifest,
```

Comment:
Is this right? comparing the same object seems weird to me but I'm not familiar with this test, just asking 🙂

Reply:
I think there's a race condition in checking the manifest here, so I'm making a hard comparison.  I actually don't think we should compare the manifests at all since it's not the change we're making in the policy itself.

What we actually need is a watcher for the manifest or a way to make sure it's actually been updated in the policy before making the assertion since it's an async operation apart from the user controlled policy update.

Reply:
Got it, so depending on how log it takes the code above, the manifest is updated with a new version. 👍